test/example.c

```diff
@@ -530,15 +644,14 @@ void test_dict_inflate(compr, comprLen, uncompr, uncomprLen)
     CHECK_ERR(err, "inflateEnd");
 
     if (strcmp((char*)uncompr, hello)) {
-        fprintf(stderr, "bad inflate with dict\n");
-        exit(1);
+        RETURN_FAILURE("bad inflate with dict\n", NULL);
     } else {
-        printf("inflate with dictionary: %s\n", (char *)uncompr);
+        RETURN_SUCCESS("inflate with dictionary: ", (char*)uncompr);
     }
 }
 
 /* ===========================================================================
- * Usage:  example [output.gz  [input.gz]]
+ * Usage:  example [--junit results.xml] [output.gz  [input.gz]]
```

Comment:
as long as we're touching this line, does example.c really take an input filename?